code/__defines/misc.dm

```diff
@@ -315,4 +315,10 @@
 
 // Defines for fluorescence (/atom/var/fluorescent)
 #define FLUORESCENT_GLOWS   1	// Glows when under flourescent light
-#define FLUORESCENT_GLOWING 2	// Currently glowing due to flourescent light
\ No newline at end of file
+#define FLUORESCENT_GLOWING 2	// Currently glowing due to flourescent light
+
+// Flags used for utensil-food interaction.
+#define UTENSIL_FLAG_COLLECT  BITFLAG(0)
```

Comment:
is there an explanation anywhere of the intended difference between "collecting" and "scooping"?